src/collections/_documentation/enriching-error-data/additional-data.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Additional Data
+sidebar_order: 0
+example_tag_name: page_locale
+example_tag_value: de-at
+example_user_email: 'john.doe@example.com'
+example_extra_key: character_name
+example_extra_value: 'Mighty Fighter'
+---
+
+Additional data attached to an event can be predefined or custom.
+
+[Predefined data](#predefined-data) is data that Sentry recognizes and uses to enhance the user experience. For example, Sentry's concept of a user lets us show how many unique users are affected by an issue. Sentry automatically captures predefined data where possible, such as information about the browser, HTTP request, and OS in the browser Javascript SDK.
```

Comment:
@MimiDumpling JavaScript, not Javascript. That's what the test is yelling about.

Reply:
Awesome! Thanks for that catch. I'll fix it shortly.